WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -279,11 +279,10 @@ - (void)followTopicNamed:(NSString *)topicName withSuccess:(void (^)())success f
         }
         return;
     }
-
     ReaderTopicServiceRemote *remoteService = [[ReaderTopicServiceRemote alloc] initWithRemoteApi:[self apiForRequest]];
-    [remoteService followTopicNamed:topicName withSuccess:^{
+    [remoteService followTopicNamed:topicName withSuccess:^(NSNumber *topicID){
         [self fetchReaderMenuWithSuccess:^{
```

Comment:
I know this was already here, but I *think* there is a possible [retain cycle](http://stackoverflow.com/questions/20030873/always-pass-weak-reference-of-self-into-block-in-arc) here.